### PR TITLE
Refactor core validate secret step

### PR DESCRIPTION
**Pull request by sirh3e**

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/tye?pullRequest=787) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/tye?pullRequest=787) before we can accept your contribution.<br/><br/>:x: sirh3e [sign now](https://cla.dotnetfoundation.org/dotnet/tye?pullRequest=787)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/tye?pullRequest=787) it.</sub>

**Comment by sirh3e** [comment]
@dnfadmin i dont get the point?

**Comment by tebeco** [comment]
(don't mind me), It's just a bot telling you that it checked, and that you properly signed the `CLA`, which are terms you must comply to, in order to contribute

**Comment by sirh3e** [comment]
@tebeco the bot name at first was not helping

**Comment by tebeco** [comment]
yeah, click on it, part of DotNetFoundation => DNF admin

File: src/Microsoft.Tye.Core/ValidateSecretStep.cs [hunk 1]
```diff
@@ -41,142 +41,142 @@ public override async Task ExecuteAsync(OutputContext output, ApplicationBuilder
 
             foreach (var binding in bindings.Bindings)
             {
-                if (binding is SecretInputBinding secretInputBinding)
+                if (!(binding is SecretInputBinding secretInputBinding))
+                    continue;
```
Comment (jkotalik) [inline src/Microsoft.Tye.Core/ValidateSecretStep.cs:45]: nit prefer braces.

**Review by jkotalik — APPROVED** [review]

**Comment by sirh3e** [comment]
Added braces

**Pull request merged**